### ISSUE TITLE
parispittman still not coming up as apple

### ISSUE THREAD

**Issue by parispittman** [cncf/gitdm.archive#396]
https://k8s.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20decade&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=Apple

there are also several people that don't work at apple on this list. 

#381 

https://k8s.devstats.cncf.io/d/55/company-prs-in-repository-groups?orgId=1&var-period_name=Last%20decade&var-repogroups=All&var-companies=Apple&var-countries=All
this also only lists my contributor experience work and doesn't have the steering repo. there are also several people on this list that are not at apple.

**Comment by lukaszgryglicki** [comment]
From #381 I've mentioned that you can create a feature request to change this, no feature request was created. I'll treat this as a feature request.
If somebody is not at Apple, then he/she should create a PR on `cncf/gitdm` removing/changing Apple affiliation. I remember there was such a PR in the past - removing *some* Apple affiliations, but finally, they figured out that they should do one more update removing others but never actually created such a PR. 

Regarding contributor experience/steering repo - is this correctly defined in repository groups definitions: https://k8s.devstats.cncf.io/d/68/repository-groups?orgId=1

**Comment by lukaszgryglicki** [comment]
I'll create a task to re-check all Apple contributors, in the meantime I'll remove the limit of minimal contributions to show on the list, that way tables will have a lot more rows, but that should work anyway. Will keep you informed.

**Comment by lukaszgryglicki** [comment]
And regarding your steering committee work: this is how steering committee is defined in DevStats:
![Screenshot 2020-07-20 at 08 20 04](https://user-images.githubusercontent.com/2469783/87906184-7367a700-ca62-11ea-8928-c4e01ca0538e.png)

I've checked GitHub on that repo:
![Screenshot 2020-07-20 at 08 25 00](https://user-images.githubusercontent.com/2469783/87906267-9e51fb00-ca62-11ea-8955-a39a317bc062.png)
Your commit was April 1st - you were not at Apple yet, your apple affiliation starts June 1st (https://github.com/kubernetes/steering/commits?author=parispittman):
```
gha=# select * from gha_actors_affiliations where actor_id = 5912992;
 actor_id | company_name |       dt_from       |        dt_to        | original_company_name | source 
----------+--------------+---------------------+---------------------+-----------------------+--------
  5912992 | AOL          | 1900-01-01 00:00:00 | 2017-06-01 00:00:00 | AOL                   | user
  5912992 | Google       | 2017-06-01 00:00:00 | 2020-06-01 00:00:00 | Google                | user
  5912992 | Apple        | 2020-06-01 00:00:00 | 2100-01-01 00:00:00 | Apple                 | user
(3 rows) 
```

Finally I've checked the events table (which records all activity):
```
gha=# select * from gha_events where dup_actor_login = 'parispittman' and dup_repo_name like '%steering%' and created_at > '2020-06-01' order by created_at desc limit 100;
       id        |       type        | actor_id |  repo_id  | public |     created_at      |  org_id  | forkee_id | dup_actor_login |    dup_repo_name    
-----------------+-------------------+----------+-----------+--------+---------------------+----------+-----------+-----------------+---------------------
     12906359555 | IssuesEvent       |  5912992 | 100529617 | t      | 2020-07-15 01:23:33 | 13629408 |           | parispittman    | kubernetes/steering
     12905386992 | IssueCommentEvent |  5912992 | 100529617 | t      | 2020-07-14 22:20:42 | 13629408 |           | parispittman    | kubernetes/steering
 281478494053586 | unassigned        |  5912992 | 100529617 | t      | 2020-07-06 19:26:39 | 13629408 |           | parispittman    | kubernetes/steering
 281478493890165 | assigned          |  5912992 | 100529617 | t      | 2020-07-06 18:34:52 | 13629408 |           | parispittman    | kubernetes/steering
 281478493890111 | subscribed        |  5912992 | 100529617 | t      | 2020-07-06 18:34:51 | 13629408 |           | parispittman    | kubernetes/steering
 281478493890107 | mentioned         |  5912992 | 100529617 | t      | 2020-07-06 18:34:51 | 13629408 |           | parispittman    | kubernetes/steering
 281478493842621 | assigned          |  5912992 | 100529617 | t      | 2020-07-06 18:19:58 | 13629408 |           | parispittman    | kubernetes/steering
     12827360395 | IssueCommentEvent |  5912992 | 100529617 | t      | 2020-07-06 18:19:45 | 13629408 |           | parispittman    | kubernetes/steering
     12827252456 | IssueCommentEvent |  5912992 | 100529617 | t      | 2020-07-06 18:06:47 | 13629408 |           | parispittman    | kubernetes/steering
     12827234185 | IssuesEvent       |  5912992 | 100529617 | t      | 2020-07-06 18:04:39 | 13629408 |           | parispittman    | kubernetes/steering
     12788370085 | IssueCommentEvent |  5912992 | 100529617 | t      | 2020-07-01 16:42:53 | 13629408 |           | parispittman    | kubernetes/steering
     12766384513 | IssuesEvent       |  5912992 | 100529617 | t      | 2020-06-29 19:28:19 | 13629408 |           | parispittman    | kubernetes/steering
 281478470903711 | labeled           |  5912992 | 100529617 | t      | 2020-06-29 19:28:19 | 13629408 |           | parispittman    | kubernetes/steering
 281478470903719 | assigned          |  5912992 | 100529617 | t      | 2020-06-29 19:28:19 | 13629408 |           | parispittman    | kubernetes/steering
 281478422217781 | assigned          |  5912992 | 100529617 | t      | 2020-06-15 17:47:06 | 13629408 |           | parispittman    | kubernetes/steering
 281478422217714 | mentioned         |  5912992 | 100529617 | t      | 2020-06-15 17:47:05 | 13629408 |           | parispittman    | kubernetes/steering
 281478422217716 | subscribed        |  5912992 | 100529617 | t      | 2020-06-15 17:47:05 | 13629408 |           | parispittman    | kubernetes/steering
 281478416775507 | assigned          |  5912992 | 100529617 | t      | 2020-06-12 22:01:44 | 13629408 |           | parispittman    | kubernetes/steering
 281478416775445 | subscribed        |  5912992 | 100529617 | t      | 2020-06-12 22:01:43 | 13629408 |           | parispittman    | kubernetes/steering
 281478416775444 | mentioned         |  5912992 | 100529617 | t      | 2020-06-12 22:01:43 | 13629408 |           | parispittman    | kubernetes/steering
 281478371201651 | subscribed        |  5912992 | 100529617 | t      | 2020-06-01 19:08:23 | 13629408 |           | parispittman    | kubernetes/steering
 281478371201648 | mentioned         |  5912992 | 100529617 | t      | 2020-06-01 19:08:23 | 13629408 |           | parispittman    | kubernetes/steering
(22 rows)
```

So yes, you have issues activity on the steering repo starting from 1st June, but no PRs, and [mentioned dashbord](https://k8s.devstats.cncf.io/d/55/company-prs-in-repository-groups?orgId=1&var-period_name=Last%20month&var-repogroups=All&var-companies=Apple&var-countries=All) shows PRs.

Summing up - I'll make the 1st dashboard showing all contributors, even with just a single contributions, I'll research all contributors assigned to Apple again - to remove incorrect affiliations. The 2nd dashboard which is not showing you is correct imho.

**Comment by lukaszgryglicki** [comment]
OK, [dashboard](https://k8s.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20quarter&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=Apple) fixed, [document](https://docs.google.com/spreadsheets/d/1WoWa07wcV_qHehgGoyZ8MaPMo6748LyXkqOd8sE874M/edit?usp=sharing) to re-check Apple contributors created.
I Will let you know when this document is checked and imported and data is regenerated.

**Comment by lukaszgryglicki** [comment]
Processed a few Apple affiliations, will now reimport data and regenerate dashboards - it should make Apple affiliations quality a bit better, but the final update will be ready in about a week (when the entire Apple affiliations spreadsheet is processed imported).

**Comment by lukaszgryglicki** [comment]
OK, imported some fixed data, it is still wip but there are a bit less Apple contributors now.

**Comment by lukaszgryglicki** [comment]
Imported the final fixed Apple data, also dashboard now shows all contributors, even with a single contribution. I think this can be closed?

**Comment by lukaszgryglicki** [comment]
I'm closing this, please reopen if needed.
